portfolio/src/main/java/com/google/sps/data/Season.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+public final class Season {
```

Comment:
This might make more sense if it were called something like "SeasonVote"?
Either way, some documentation would go a long way here. Either some comments above the class or maybe some JavaDoc?